### DISCUSSION TITLE
Formulario de entrega cerrado

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#237] [Preguntas y respuestas] [answered]
Chicos tengo preguntarles algo importante. No esta disponible el formulario de entrega. Saben que pasa? es que ya terminamos el TPI y falta un dia para entregarlo. Me pueden decir que hago

**Comment by martinvilu** [comment, marked as answer]
Vamos a revisar los trabajos integradores todos juntos el dia lunes, no hace falta entregar.
